src/background/RemotePontoon.ts

```diff
@@ -195,45 +203,75 @@ async function updateLatestTeamActivity() {
   }
 }
 
+async function updateTeam(): Promise<StorageContent['team']> {
+  const teamCode = await getOneOption('locale_team');
+  const [pontoonData, bugzillaComponentsResponse] = await Promise.all([
+    pontoonRestClient.getTeamInfo(teamCode),
+    httpClient.fetch(bugzillaTeamComponents()),
+  ]);
+  const bugzillaComponents = (await bugzillaComponentsResponse.json()) as {
+    [code: string]: string;
+  };
+
+  const team: StorageContent['team'] = {
+    code: pontoonData.code,
+    name: pontoonData.name,
+    strings: {
+      approvedStrings: pontoonData.approved_strings,
+      pretranslatedStrings: pontoonData.pretranslated_strings,
+      stringsWithWarnings: pontoonData.strings_with_warnings,
+      stringsWithErrors: pontoonData.strings_with_errors,
+      missingStrings: pontoonData.missing_strings,
+      unreviewedStrings: pontoonData.unreviewed_strings,
+      totalStrings: pontoonData.total_strings,
+    },
+    bz_component: bugzillaComponents[pontoonData.code],
+  };
+
+  await saveToStorage({ team });
+  return team;
+}
 async function updateTeamsList(): Promise<StorageContent['teamsList']> {
   const [pontoonData, bugzillaComponentsResponse] = await Promise.all([
-    pontoonGraphqlClient.getTeamsInfo(),
+    pontoonRestClient.getTeamsInfo(),
     httpClient.fetch(bugzillaTeamComponents()),
   ]);
   const bugzillaComponents = (await bugzillaComponentsResponse.json()) as {
     [code: string]: string;
   };
-  const sortedTeams = pontoonData.locales
-    .filter((team) => team.totalStrings > 0)
+  const sortedTeams = pontoonData
+    .filter((team) => team.total_strings > 0)
     .sort((team1, team2) => team1.code.localeCompare(team2.code));
   const teamsList: StorageContent['teamsList'] = {};
+
   for (const team of sortedTeams) {
     teamsList[team.code] = {
       code: team.code,
       name: team.name,
       strings: {
-        approvedStrings: team.approvedStrings,
-        pretranslatedStrings: team.pretranslatedStrings,
-        stringsWithWarnings: team.stringsWithWarnings,
-        stringsWithErrors: team.stringsWithErrors,
-        missingStrings: team.missingStrings,
-        unreviewedStrings: team.unreviewedStrings,
-        totalStrings: team.totalStrings,
+        approvedStrings: team.approved_strings,
+        pretranslatedStrings: team.pretranslated_strings,
+        stringsWithWarnings: team.strings_with_warnings,
+        stringsWithErrors: team.strings_with_errors,
+        missingStrings: team.missing_strings,
+        unreviewedStrings: team.unreviewed_strings,
+        totalStrings: team.total_strings,
```

Comment:
We should not be storing these stats for all locales; all we really need from here are the `code` and the `name`, and doing a `getTeamInfo()` if/when the selected locale changes. See comment below on `getTeamsInfo()`.